ingest-app/docs/api.md

```diff
@@ -280,6 +284,13 @@ A native-id is an identifier, unique per provider, used to identify concepts wit
 
 ## <a name="metadata-ingest"></a> Metadata Ingest
 
+### <a name="latest-umm-versions"></a> Latest UMM Schema Versions
+
+The following are the latest acceptable UMM schema versions for metadata ingest:
+
+<p>UMM-C: {{ umm-c }}<br/>UMM-G: {{ umm-g }}<br/>UMM-S: {{ umm-s }}<br/>UMM-T: {{ umm-t }}<br/>UMM-SUB: {{ umm-sub }}<br/>UMM-VAR: {{ umm-var }}</p>
```

Comment:
Nitpick: we could just do this an an unordered list:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li
```suggestion
<ul class="umm-versions">
  <li>UMM-C: {{ umm-c }}</li>
  <li>UMM-G: {{ umm-g }}</li>
  <li>UMM-S: {{ umm-s }}</li>
  <li>UMM-T: {{ umm-t }}</li>
  <li>UMM-SUB: {{ umm-sub }}</li>
  <li>UMM-VAR: {{ umm-var }}</li>
</ul>
```